master/internal/experiment/bulk_action.go

```diff
@@ -60,6 +54,18 @@ func terminalExperimentStates() []string {
 	return states
 }
 
+func nonTerminalStates() []experimentv1.State {
```

Comment:
i would define this as `var NonTerminalStates []State` in the `model` package and separately add a to proto for a list of states.

e.g.,
```
var NonTerminalStates = func() []State {
	var states []State
	for s := range model.ExperimentTransitions {
		if model.TerminalStates[s] {
			continue
		}

		states = append(states, s)
	}
	return states
}()
```